contract-tests/tests/test/amazon/botocore/botocore_test.py

```diff
@@ -58,7 +58,7 @@ def get_application_image_name() -> str:
     def set_up_dependency_container(cls):
         local_stack_networking_config: Dict[str, EndpointConfig] = {
             NETWORK_NAME: EndpointConfig(
-                version="1.22",
+                version="1.25",
```

Comment:
Not sure if these versions need to be in sync with the rest of the version updates, but did it anyway.

Reply:
These are unrelated. Actually prefer you leave them as is, to avoid confusion down the road.

Reply:
Got it. Will revert.